config/rbac/role.yaml

```diff
@@ -18,6 +19,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ocs.openshift.io
+  resources:
+  - managedocs/finalizers
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Can we fold it into the other instance of the `ocs.openshift.io` group?

Reply:
Done